martin-ui/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    // assets can also be the name of a tile source
+    // so we use _assets to avoid conflicts
```

Comment:
Looking at all the requests when viewing `localhost:3000/` while setting `RUST_LOG=debug`, I see these requests:

```
[2024-07-09T14:32:23Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36" 0.000332
[2024-07-09T14:32:23Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /assets/index-0vhmA-Yl.js HTTP/1.1" 304 0 "http://0.0.0.0:3000/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36" 0.000802
[2024-07-09T14:32:23Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /_assets/index.RjhygAlW.css HTTP/1.1" 304 0 "http://0.0.0.0:3000/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36" 0.001011
[2024-07-09T14:32:23Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /_assets/logo.ZTiqb92t.png HTTP/1.1" 304 0 "http://0.0.0.0:3000/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36" 0.000364
```

So it clearly gets BOTH `/assets` and `/_assets` -- not good. Moreover, it IS possible to create an `_assets` source, so it could conflict.  The only way to NOT have any conflicts is to add all assets to `/_/assets/*` path -- this way it will never conflict (the `_` is a reserved keyword)

Reply:
I'm not sure that this PR is complete. When I run `npm run build && npm run dev` in the `martin-ui` sub-directory, the UI runs fine. When I run `cargo run --  ./world_cities.mbtiles`, local is unable to find the js file at the root, but is able to find a css file at /assets. I'm not sure what is happening at `cargo run ...` that would prevent it from finding the js file.

Reply:
why would anything access things at `/assets/...`?

Reply:
I think that my latest change meets what you are looking for with js, css and image files all served from `_/assets`. Running `cargo run --  ./world_cities.mbtiles` now runs the web UI at `/`. 

If it doesn't, can you outline the directory or output build structure you have in mind?